app/experimenter/experiments/bugzilla.py

```diff
@@ -14,18 +14,22 @@ class BugzillaError(Exception):
 
 def format_bug_body(experiment):
     bug_body = ""
-    countries = "".join(
-        [
-            "{name} ({code}) ".format(name=country.name, code=country.code)
-            for country in list(experiment.countries.all())
-        ]
-    )
-    locales = "".join(
-        [
-            "{name} ({code}) ".format(name=locale.name, code=locale.code)
-            for locale in list(experiment.locales.all())
-        ]
-    )
+    countries = "all"
+    locales = "all"
+    if len(experiment.countries.all()) > 0:
```

Comment:
Nit: you can use https://docs.djangoproject.com/en/2.2/ref/models/querysets/#count for this